code/modules/mob/living/carbon/xenomorph/xenoprocs.dm

```diff
@@ -756,3 +758,12 @@
 
 /mob/living/carbon/xenomorph/proc/hit_and_run_bonus(damage)
 	return damage
+
+/mob/living/carbon/xenomorph/proc/is_burrowed_larva_host()
```

Comment:
yep as i thought, this is what i've been working hard to get rid of, dont define this stuff at xeno level, use signals or define it only on types that need it